{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1729197488641,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win10-COMPILED_BY-vs2022 #30892","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/30892/","buildNumber":30892,"builtOn":"test-azure_msft-win10_vs2019-x64-3","nodeVersion":"23.0.0","commitHash":"0676a6df6d96724580bf56d0a6d5090ce188e71f","callStack":[{"upstreamBuild":66593,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":75285,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":63166,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"55192"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":293,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (1122.5795ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (1182.8549ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (1251.5549ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (1795.8253ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (1104.9069ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (1153.369ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (2231.7579ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (1249.9361ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (1192.1667ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (1273.0291ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (1072.614ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (1347.7411ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (1343.3257ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (845.5788ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (1135.6698ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (832.6849ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (974.2637ms)\n  ✖ parallel\\test-async-local-storage-bind.js (865.0619ms)\n    AssertionError [ERR_ASSERTION]: Test parallel\\test-async-local-storage-bind.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:935:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ parallel\\test-async-local-storage-contexts.js (937.6448ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (1336.5401ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (923.5327ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (1100.0378ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (1037.3003ms)\n✖ AsyncContextFrame (27320.7323ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 27328.7084\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:49:5\n✖ parallel\\test-async-local-storage-bind.js (865.0619ms)\n  AssertionError [ERR_ASSERTION]: Test parallel\\test-async-local-storage-bind.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:935:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":27831.066,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}